src/integrationTest/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/resource/UpdateInductionTest.kt

```diff
@@ -544,7 +544,7 @@ class UpdateInductionTest : IntegrationTestBase() {
       .hasPrisonId("BXI")
       .wasModifiedBy("auser_gen")
       .wasCreatedAt(persistedInduction.createdDateTime)
-      .wasLastModifiedAt(persistedInduction.modifiedDateTime) // should be unchanged
+      .wasLastModifiedAfter(persistedInduction.modifiedDateTime)
```

Comment:
Due to some logic within `CiagInductionResponseMapper` (see below), if we weren't updating the `workOnRelease` object (e.g. by switching from long route to short route), then I expected the `lastModified` date to stay the same.

Coming back to it now, this seems "fundamentally wrong" and I think it makes much more sense to ensure the main Induction's timestamp etc is updated each time an induction is changed.